Exercises/01/Begin/CMakeLists.txt

```diff
@@ -1,7 +1,16 @@
 cmake_minimum_required(VERSION 3.16)
 project(KokkosTutorial01)
-include(../../common.cmake)
+
+# Add a custom module path for find_package
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../.cmake)
```

Comment:
I think you are missing quotes around ${CMAKE_CURRENT_SOURCE_DIR}/../../.cmake. They are present in all other CMakeLists.txt you modified.